library/std/src/sys/pal/unix/thread.rs

```diff
@@ -225,9 +225,12 @@ impl Thread {
         // Newlib, Emscripten, and VxWorks have no way to set a thread name.
     }
 
-    #[cfg(target_os = "linux")]
+    #[cfg(any(target_os = "linux", target_os = "freebsd", target_os = "netbsd",))]
     pub fn get_name() -> Option<CString> {
+        #[cfg(any(target_os = "linux", target_os = "freebsd"))]
         const TASK_COMM_LEN: usize = 16;
```

Comment:
I think this is [defined](https://github.com/freebsd/freebsd-src/blob/9241ebc796c11cf133c550f188f324bd2c12d89a/sys/compat/linuxkpi/common/include/linux/sched.h#L64) as `MAXCOMLEN + 1 == 20` on FreeBSD.
```suggestion
        #[cfg(target_os = "linux")]
        const TASK_COMM_LEN: usize = 16;
        #[cfg(target_os = "freebsd")]
        const TASK_COMM_LEN: usize = libc::MAXCOMLEN + 1;
```

Reply:
ah true I missed it